runtime/server/server.go

```diff
@@ -239,13 +239,19 @@ func HTTPErrorHandler(ctx context.Context, mux *gateway.ServeMux, marshaler gate
 	gateway.DefaultHTTPErrorHandler(ctx, mux, marshaler, w, r, err)
 }
 
-func timeoutSelector(service, method string) time.Duration {
-	if service == "rill.runtime.v1.RuntimeService" && (strings.Contains(method, "Trigger") || strings.Contains(method, "Reconcile")) {
+func timeoutSelector(fullMethodName string) time.Duration {
+	if strings.HasPrefix(fullMethodName, "rill.runtime.v1.RuntimeService") && (strings.Contains(fullMethodName, "/Trigger") || strings.Contains(fullMethodName, "/Reconcile")) {
```

Comment:
1. Missing `/` prefix for service name check
2. I think some of the reconcile functions current aren't prefixed with `Reconcile`, so probably shouldn't have `/` prefix